sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecutionRDD.scala

```diff
@@ -39,7 +39,7 @@ class SQLExecutionRDD(
   private val sqlConfigs = conf.getAllConfs
   private lazy val sqlConfExecutorSide = {
     val props = new Properties()
-    props.putAll(sqlConfigs.asJava)
+    sqlConfigs.foreach { case (k, v) => props.setProperty(k, v) }
```

Comment:
I know it is a bit late, but why are we not setting the configurations directly?

Reply:
@mgaido91 any idea?

Reply:
the reason why I created the properties object was to have it in a synchronized block, but since we are in a lazy val it is probably useless...

Reply:
@mgaido91 can you send a followup PR to clean it up? Now we can't set the SQL confs in a batch and I don't think it's useful to have this `Properties`.

Reply:
@hvanhovell and @mgaido91 . Feel free to make a follow-up if we need. Also, please apply that in the backporting PR, #25734, before merging for consistency.

Reply:
sure @cloud-fan , I am doing that. Thanks for the remainder @dongjoon-hyun, I'll update the backport PR once this is stable. Thanks.